### ISSUE TITLE
exterior LCD display on board

### ISSUE THREAD

**Issue by tsaoyu** [Maritime-Robotics-Student-Society/sailing-robot#145]
![](http://files.hwkitchen.com/200000137-cf454cffdd/AMC2004A-B-Y6WFDY_arduino_10.JPG)

A display like this with information like:
* Name of current launch file
* Power level of battery
* Autonomous/Manual mode

**Comment by Nanoseb** [comment]
Haha, the launch file name, good idea ;)

**Comment by smaria** [comment]
This looks like it will be a mess during waterproofing and will very likely be covered up by bumblebee tape in most use cases. How about we display the name of the launch file on the dashboard? (same goes for battery state, once we can read it)

**Comment by takluyver** [comment]
I agree that info on the dashboard is more practical use in most cases, and less of a concern for waterproofing.

Do we know of a way to get the name of the launch file from inside ROS? My searches for that are not turning up anything.
